pkg/kubectl/cmd/create_service.go

```diff
@@ -80,6 +80,7 @@ func NewCmdCreateServiceClusterIP(f cmdutil.Factory, cmdOut io.Writer) *cobra.Co
 	cmdutil.AddGeneratorFlags(cmd, cmdutil.ServiceClusterIPGeneratorV1Name)
 	addPortFlags(cmd)
 	cmd.Flags().String("clusterip", "", i18n.T("Assign your own ClusterIP or set to 'None' for a 'headless' service (no loadbalancing)."))
+	cmd.Flags().StringP("selector", "l", "", "Label Selector to define set of pods targeted by service, supports only '='.")
```

Comment:
Do we use that language `supports only '='` other places? It confused me. I thought you were saying the flag had to be `--selector=blah` or `-l=blah` instead of `--selector blah` or `-l blah`. What do we do in other places?